packages/loader/driver-utils/src/treeConversions.ts

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Uint8ArrayToString,unreachableCase} from "@fluidframework/common-utils";
+import { AttachmentTreeEntry, BlobTreeEntry, TreeTreeEntry } from "@fluidframework/protocol-base";
+import * as api from "@fluidframework/protocol-definitions";
```

Comment:
```suggestion
import { ISummaryTree, ITree, ITreeEntry, SummaryType } from "@fluidframework/protocol-definitions";
```
Maybe it would be better to only import the used interfaces/types?